app/controllers/main_controller.rb

```diff
@@ -10,7 +10,7 @@ def search
     minimum_query_length = 3
 
     if params[:query] and params[:query].length >= minimum_query_length
-      @activities = Activity.includes(:user).where("content LIKE ?", "%#{params[:query]}%").limit(params[:limit] || 100).all
+      @activities = Activity.includes(:user).where("LOWER(content) LIKE ?", "%#{params[:query]}%").limit(params[:limit] || 100).all
```

Comment:
Really, we should be doing a case insensitive search, as if we are looking for "Blah" but there was an instance of "blah", then we wouldn't catch it.